string name
string uuid
string connection_type
string wifi_ssid
string wifi_security

string ETHERNET_CONNECTION=Ethernet
string WIFI_CONNECTION=802.11 WiFi
string OTHER_CONNECTION=Other